zetasql/parser/bison_parser.y

```diff
@@ -2366,6 +2417,28 @@ opt_as_sql_function_body_or_string:
       }
     ;
 
+identifier_or_string_list:
+    identifier_or_string
```

Comment:
I think simple **identifier** is enough, no need to support string as well.

Reply:
we may support IP control as 'user'@'IP' later. so users may create by strings as 
`create user 'user1';`

Reply:
I think all other thing can left to `options_list` like discussed below